<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SquareCreation</class>
 <widget class="QWidget" name="SquareCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="spinBox_cornerX">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_cornerY">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_corner">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Левый нижний угол квадрата</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>215</width>
     <height>69</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_randomise">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_randomMin">
      <item>
       <widget class="QLabel" name="label_randomMin">
        <property name="text">
         <string>Нижняя граница случайных значений</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_randomMin">
        <property name="minimum">
         <number>-99</number>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_randomise">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="recources.qrc">
        <normaloff>:/D:/Users/user/Downloads/dice.png</normaloff>:/D:/Users/user/Downloads/dice.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_randomMax">
      <item>
       <widget class="QLabel" name="label_randomMax">
        <property name="text">
         <string>Верхняя граница случайных значений</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_randomMax">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_cornerBrackets">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(          ,          )</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_side">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сторона квадрата</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_side">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
